ocaml/xapi/pool_recommendations.ml

```diff
@@ -0,0 +1,41 @@
+(*
+ * Copyright (c) Cloud Software Group, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+module Unixext = Xapi_stdext_unix.Unixext
+module Config_file = Xcp_service.Config_file
+
+module D = Debug.Make (struct let name = "pool_recommendations" end)
+
+open D
+module StringMap = Map.Make (String)
```

Comment:
The operation on this data structure is only `add`. No `find`, `update`, etc.
I understand a `list` would work as well.